CameraInspector/Frame.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+namespace CameraInspector
+{
+
+class Frame
+{
+public:
+	Frame() = default;
+	//Mat(Size size, int type, void *data, size_t step = AUTO_STEP)
+	Frame(int height, int width, void* data);			// QImage|cv::Mat to frame
+	
+	void Construct(int height, int width, void* data);
+
+	int GetRows() const;
+	int GetCols() const;
+	void* GetData() const;
+
+private:
+	int cols;			// width
```

Comment:
Fixed: 2e4ec15
I think it would be more informative to think about frame resolution as columns and rows. Instead, QImage has width and height. So I prefered remove comments to change variable names